colors/dracula.vim

```diff
@@ -53,6 +53,20 @@ let s:yellow    = ['#F1FA8C', 228]
 
 let s:none      = ['NONE', 'NONE']
 
+let g:dracula_palette = {
```

Comment:
This will be super useful for other plugin integrations (looking at you, airline). Good call.

Reply:
I think so to, thanks. I ended up adding in the extended colors as well. Figured some other UI themes might want to use those as well. :+1: